model/model/v1alpha/model.proto

```diff
@@ -1773,8 +1773,8 @@ message ModelRun {
   // Requester ID. This field might be empty if the model run belongs to a
   // deleted namespace.
   string requester_id = 16 [(google.api.field_behavior) = OUTPUT_ONLY];
```

Comment:
Could you take the opportunity to document the difference between requester and runner? 🙏

Reply:
updated